spec/intl.html

```diff
@@ -159,13 +159,27 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
         1. <del>Else,</del>
           1. <del>Set _dateTimeFormat_.[[HourCycle]] to *undefined*.</del>
           1. <del>Let _pattern_ be _bestFormat_.[[pattern]].</del>
-        1. Set _dateTimeFormat_.[[Pattern]] to <del>_pattern_</del><ins>GetDateTimeFormatPattern(_opt_, _formats_, _hc_)</ins>.
+        1. <ins>Let _expandedOptions_ be a copy of _opt_.</ins>
+        1. <ins>Let _needDefaults_ be *true*.</ins>
+        1. <ins>For each _field_ of « `"weekday"`, `"year"`, `"month"`, `"day"`, `"hour"`, `"minute"`, `"second"` », do</ins>
+          1. <ins>If _expandedOptions_.[[&lt;_field_>]] is not *undefined*, then</ins>
+            1. <ins>Set _needDefaults_ to *false*.</ins>
+        1. <ins>If _needDefaults_ is *true*, then</ins>
+          1. <ins>For each _field_ of « `"year"`, `"month"`, `"day"` », do</ins>
```

Comment:
I believe the most recent discussion with @sffc in NY concluded that the default should be year, month, day, hour, minute, second all being numeric if nothing is defined.